hedera-node/hedera-app/src/main/java/com/hedera/node/app/store/ServiceApiFactory.java

```diff
@@ -36,8 +38,8 @@ public class ServiceApiFactory {
     private final Configuration configuration;
     private final StoreMetricsService storeMetricsService;
 
-    private static final Map<Class<?>, ServiceApiProvider<?>> API_PROVIDER =
-            Map.of(TokenServiceApi.class, TOKEN_SERVICE_API_PROVIDER);
+    private static final Map<Class<?>, ServiceApiProvider<?>> API_PROVIDER = Map.of(
+            TokenServiceApi.class, TOKEN_SERVICE_API_PROVIDER, ScheduleServiceApi.class, SCHEDULE_SERVICE_API_PROVIDER);
```

Comment:
I don't see a reason to add this `ScheduleServiceApi.class`...may be missing something? 🤔